frontend/components/bridge/DepositForBridging.tsx

```diff
@@ -212,10 +220,26 @@ export const DepositForBridging = ({
     data: bridgeFundingRequirements,
     isLoading: isBridgeRefillRequirementsLoading,
     isError: isBridgeRefillRequirementsError,
+    isFetching: isBridgeRefillRequirementsFetching,
+    refetch: refetchBridgeRefillRequirements,
   } = useBridgeRefillRequirements(bridgeRequirementsParams);
 
+  // fetch bridge refill requirements manually on mount
+  useEffect(() => {
+    if (!isBridgeRefillRequirementsApiLoading) return;
+
+    refetchBridgeRefillRequirements().finally(() => {
+      setIsBridgeRefillRequirementsApiLoading(false);
+    });
+  }, [
+    isBridgeRefillRequirementsApiLoading,
+    refetchBridgeRefillRequirements,
+    setIsBridgeRefillRequirementsApiLoading,
+  ]);
+
```

Comment:
why?

Reply:
can't we use isLoading for this purpose? afair it's true on first load and then if data is chached it jusr refetched and sets isPending to true instead